contracts/contracts/lib/AdditionalMath.sol

```diff
@@ -3,14 +3,13 @@
 pragma solidity ^0.8.0;
 
 
-import "../../zeppelin/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
 
 
 /**
 * @notice Additional math operations
 */
 library AdditionalMath {
```

Comment:
Looks like after removing some contract `AdditionalMath` is also deprecated. Would be great if we can delete too

Reply:
removed